### PR TITLE
FPO-45: Accuracy reporting

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

FPO-45

### What?

I have added/removed/altered:

- [x] Adds benchmark Makefile targets for RIS and HMRC data sources
- [x] Update benchmark script to incorporate cleaning of data sources
- [x] Adds initial benchmark build thresholds
- [x] Adds benchmarking step to model generation
- [x] Adds script to fetch the latest versions for comparison
- [ ] Adds checking script to validate the change in the model and fail the build

### Why?

I am doing this because:

- This is a prerequisite for CD of models
- This enables us to be able to graph up the accuracy change of the models over time

### AC

- We're making decisions about whether to release based on benchmark results
- We have files in s3 for every model
- We can rerun the benchmarks and get the same results for our derivation and model combo

### CODE REVIEW

**Review by amberstarlight — APPROVED** [review]

File: training/train_model.py [hunk 1]
```diff
@@ -47,9 +52,10 @@ def run(
 
         batches = len(train_loader)
 
+        current_loss = None
```
Comment (gurdipk) [inline training/train_model.py:55]: can set to current_loss = 0
